src/Microsoft.Sbom.Api/SBOMGenerator.cs

```diff
@@ -94,79 +96,47 @@ public SBOMGenerator(StandardKernel kernel, IFileSystemUtils fileSystemUtils)
                 throw new ArgumentException($"'{nameof(rootPath)}' cannot be null or whitespace.", nameof(rootPath));
             }
 
-            if (files is null)
-            {
-                throw new ArgumentNullException(nameof(files));
-            }
+            ArgumentNullException.ThrowIfNull(files);
```

Comment:
It seems like code in this API method is almost similar to the override above. Is the difference intentional? If not, would it be reasonable to extract shared code into separate private method to make sure core logic is the same across argument overrides?